robottelo/ui/base.py

```diff
@@ -193,8 +193,8 @@ def delete_entity(self, name, really, name_locator, del_locator,
             element = self.wait_until_element((strategy1, value1 % name))
             if element is None:
                 raise UINoSuchElementError(
-                    'Could not select the entity "{0}" for deletion.'
-                    .format(name)
+                    'Could not select the entity 'u"{0}"' for deletion.'
+                    .format(u'name')
```

Comment:
This will format with plain "name" and not the content of the name variable. Should be changed back to `.format(name)`.